packages/runner/src/iframe/iframes.jsx

```diff
@@ -170,19 +170,19 @@ export default class Iframes extends Component {
     return $autIframe
   }
 
-  _addCrossDomainIframe = (location) => {
+  _addCrossOriginIframe = (location) => {
```

Comment:
For each domain established with `cy.domain` are we opening a new iframe and hiding the previous one?